include/falcosecurity/internal/symbols_extraction.h

```diff
@@ -29,9 +29,11 @@ limitations under the License.
     static plugin_mixin<__t> s_plugin_extraction;                              \
                                                                                \
     FALCOSECURITY_EXPORT                                                       \
-    uint16_t* plugin_get_extract_event_types(uint32_t* numtypes)               \
+    uint16_t* plugin_get_extract_event_types(uint32_t* numtypes,               \
```

Comment:
Shouldn't this apply to `symbols_parsing.h` as well?

Reply:
Yep! Forgot about that